lib/streams/adminLookupStream.js

```diff
@@ -0,0 +1,19 @@
+var through = require( 'through2' );
+var logger = require( 'pelias-logger' ).get( 'openaddresses' );
+var peliasAdminLookup = require( 'pelias-wof-admin-lookup' );
+
+function createAdminLookupStream(enable, peliasConfig) {
+  if (enable) {
+    logger.info( 'Setting up admin value lookup stream.' );
```

Comment:
here you log inside the `if`, but for deduper outside. should probably be consistent. i like having the log statement inside the `if` so it's muted when disabled.